packages/sdk-core/src/modules/attribute/AttributeManager.ts

```diff
@@ -30,11 +26,11 @@ export class AttributeManager {
      */
     public async addProvider(attributeProvider: BacktraceAttributeProvider) {
```

Comment:
actually maybe we can just keep add instead of addProvider and move the addprovider implementation to add ?

Reply:
I would keep them separate. `add` could be used to just add attributes or function resolving to attributes, and `addProvider` will just accept `BacktraceAttributeProvider`.